test/observer.test.tsx

```diff
@@ -564,6 +564,20 @@ it("should only called new Reaction once", () => {
     spy.mockRestore()
 })
 
+it("should support hoisting", () => {
```

Comment:
```suggestion
it("should hoist known statics only", () => {
```

Reply:
Also, add `propTypes` to the test to make the coverage happier :)

Reply:
Done